e2e-tests/playwright/e2e/plugins/acr.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { test, chromium, firefox } from '@playwright/test';
+import { UIhelper } from '../../utils/UIhelper';
+import { Common } from '../../utils/Common';
+
+let page;
+test.describe('Test ACR plugin', () => {
+  let uiHelper: UIhelper;
+  let common: Common;
+
+  test.beforeAll(async ({ browserName }) => {
+    const browserType = browserName === 'firefox' ? firefox : chromium;
+    const browser = await browserType.launch();
+    page = await browser.newPage();
+
+    uiHelper = new UIhelper(page);
+    common = new Common(page);
+    await common.loginAsGuest();
+  });
+
+  test('Verify ACR Images are visible', async () => {
+    await uiHelper.openSidebar('Catalog');
+    await uiHelper.verifyHeading('My Org Catalog');
+    await uiHelper.selectMuiBox('Kind', 'component');
+    await uiHelper.clickLink('acr-test-entity');
+    await uiHelper.clickTab('Image Registry');
+    await uiHelper.verifyHeading(
+      'Azure Container Registry Repository: hello-world',
+    );
+    await uiHelper.verifyRowInTableByUniqueText('latest', [
+      'Jan 18, 2024, 8:26 PM',
```

Comment:
I believe setting this date here makes our test quite fragile and prone to breaking, as the date could be overwritten over time. I have a test that checks the date, where I use regex to verify if the field itself could be a date. This is just a suggestion.

Reply:
updated